e2e/artefacts/k8s/vsp-ds.yaml

```diff
@@ -33,8 +33,13 @@ spec:
         volumeMounts:
         - name: vendor-plugin-sock
           mountPath: /var/run/dpu-daemon/
+        - name: dbus-socket
+          mountPath: /var/run/dbus
       volumes:
       - name: vendor-plugin-sock
         hostPath:
           path: /var/run/dpu-daemon/
+      - name: dbus-socket
+        hostPath:
+          path: /var/run/dbus/
```

Comment:
Why do we need this?

Reply:
Networkmanager daemon runs on host and exports net interface info to clients using dbus. nmcli package installed within container....needs the dbus volume mounted, so it can get network interface info.